refactoring/initial_global.py

```diff
@@ -50,12 +50,12 @@ def add_person(name, age, job, relations):
     "Zalika": "landlord"
 }
 
-add_person("Nash", 34, "chef", nash_relations)
+add_person(group, "Nash", 34, "chef", nash_relations)
 
-forget("Nash", "John")
+forget(group, "Nash", "John")
 
 if __name__ == "__main__":
     assert len(group) == 4, "Group should have 4 members"
```

Comment:
Really good, you could also only define the group within the `if __name__ == "__main__"` block, so that you don't have a group defined if you import this file